01-java-helloworld/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+    java
+    application
```

Comment:
Add and configure the following plugins:
* https://github.com/DanySK/git-sensitive-semantic-versioning-gradle-plugin
* https://github.com/JLLeitschuh/ktlint-gradle
* de.fayard.buildSrcVersions
* com.github.spotbugs
* pmd
* checkstyle

Use the alchemist build.gradle.kts for reference and configuration. In case of problems, I can help.